rover-apps/science/src/AdafruitSTEMMAModule.cpp

```diff
@@ -0,0 +1,12 @@
+#include "AdafruitSTEMMAModule.h"
+#include "Module.h"
+#include "mbed.h"
+#include "AdafruitSTEMMA.h"
+
+AdafruitSTEMMAModule::AdafruitSTEMMAModule() {
+    bool resetStatus = moistureSensor.reset();
```

Comment:
For the constructor of the module,  I think the optimal thing is to reset it to original state. Although, I'm not sure what should be done if the status returns false.

Reply:
It should log an error. Or you can keep retrying to `reset()` for a certain number of tries, then log an error if all of them failed.